dotnet/build-publish-zip.ps1

```diff
@@ -3,14 +3,15 @@ param (
     [string]$RepoName,
     [Parameter(Mandatory=$true)]
     [string]$Version,
-    [string]$Project = "."
+    [string]$Project = ".",
+    [string]$Platform = "-r win-x64"
 )
 
 $packagesDir = New-Item -ItemType directory -Path package -Force
 
 Push-Location $RepoName
 try {
-    dotnet publish $Project --nologo --sc -c Release -r win-x64 -o publish /p:Version=$Version || $(throw "dotnet publish failed")
+    dotnet publish $Project --nologo --sc -c Release $Platform -o publish /p:Version=$Version || $(throw "dotnet publish failed")
```

Comment:
This might work on Windows, but shouldn't on Linux/macOS, because `$Platform` will be expanded as a single argument with a space in it, not as two separate artifacts. I'd suggest using:
```powershell
-r $Platform
```
instead.

Reply:
Yeah okay, this makes sense. I have changed this now.